examples/cpp_exec/serialized_fn/serialize.py

```diff
@@ -0,0 +1,45 @@
+r"""Sample code to show how to serialize and deserialize a JAX executable.
+
+To invoke:
+  $ bazel run -c opt examples/cpp_exec/serialized_fn:serialize -- /tmp/serialized_fn
+"""
+
+import jax
+from jax._src import xla_bridge as xb
+import jax.numpy as jnp
+import sys
+
+
+@jax.jit
+def my_function(x, y):
+  return x + y
+
+
+def main(argv):
+  lowered = jax.jit(my_function).lower(1.0, 2.0)
+
+  compiled = lowered.compile()
+
+  u_compiled = compiled._executable._unloaded_executable
```

Comment:
I don't think we should have examples where we show people how to access internal APIs :)

Reply:
Is there a way to dump serialized function without access these? Or you are suggesting us make some changes to allow it? Context is that there have requests to export the compiled functions and execute them in cpp or Rust.

Reply:
The pickle version is not what folks need. The idea is to run the function in pure C++ (without py involvement).

Reply:
Then we need to expose an API for this.

Reply:
Thanks, added the api in https://source.corp.google.com/piper///depot/google3/third_party/py/jax/experimental/serialize_executable.py. PTAL.

Reply:
Let's add that in a separate CL please with a more descriptive description. I am sure others in the team would have opinions :)